crates/astria-sequencer/src/proposal/block_size_constraints.rs

```diff
@@ -0,0 +1,96 @@
+use anyhow::anyhow;
+
+/// The maximum number of bytes allowed in sequencer action data.
+const MAX_SEQUENCE_DATA_BYTES_PER_BLOCK: usize = 256_000;
+
+/// Struct for organizing block size constraints in prepare proposal
+pub(crate) struct BlockSizeConstraints {
+    pub(crate) max_sequencer: usize,
+    pub(crate) max_cometbft: usize,
+    pub(crate) current_sequencer: usize,
+    pub(crate) current_cometbft: usize,
+}
+
+impl BlockSizeConstraints {
+    pub(crate) fn new(cometbft_max_size: usize) -> Self {
+        BlockSizeConstraints {
+            max_sequencer: MAX_SEQUENCE_DATA_BYTES_PER_BLOCK,
+            max_cometbft: cometbft_max_size,
+            current_sequencer: 0,
+            current_cometbft: 0,
```

Comment:
Set this to `current_cometbft: GeneratedCommitments::TOTAL_SIZE` and also provide an associated const on it:
```rust
impl GeneratedCommitments {
    const TOTAL_SIZE: usize = 64; // or whatever it was
}
```